src/Integration.Vsix/SonarLintDaemon/DaemonCommands.vsct

```diff
@@ -2,11 +2,11 @@
 <CommandTable xmlns="http://schemas.microsoft.com/VisualStudio/2005-10-18/CommandTable" xmlns:xs="http://www.w3.org/2001/XMLSchema">
 
   <Commands package="guidDaemonPackagePkgString">
-    <Groups>
+    <!--<Groups>
```

Comment:
is that commented out on purpose?

Reply:
Oops, no. It's probably stopped the "Disable SonarLint rule" command from appearing...